### ISSUE TITLE
Error when running ADE20k pre-trained model

### ISSUE THREAD

**Issue by jijeng** [microsoft/CoCosNet#12]
Thanks for your wonderful work. I want to use your pre-trained model on my datasets. 
using this command

`python test.py --name ade20k --dataset_mode ade20k --dataroot ./imgs/ade20k --gpu_ids 0 --nThreads 0 --batchSize 6 --use_attention --maskmix --noise_for_mask --warp_mask_losstype direct --PONO --PONO_C --which_epoch 90`

I thought  it was not necessary to use ref mask when using a pre-trained model, so I removed all the **.png** images in `imgs/ade20k`. But there was an error

![](https://files.catbox.moe/6jjp40.jpg)

I wonder if I can use your pre-trained model if I don't have segmentation images ( masking images), thanks

**Comment by panzhang0212** [comment]
If you removed all the .png images in imgs/ade20k, how do you give the input mask(not ref mask) to network, which makes len(self.label_paths) = 0 , and raise out range error.

**Comment by jijeng** [comment]
if I removed the .png in `ade20k/training` or  `ade20k/validation`, then there also some errors.


![image](https://user-images.githubusercontent.com/24850280/91157829-6b052a80-e6f8-11ea-83ba-7a0aca26a040.png)

![image](https://user-images.githubusercontent.com/24850280/91158031-aa337b80-e6f8-11ea-9835-972b1ba5acc0.png)

**Comment by panzhang0212** [comment]
I mean you couldn't delete all .png image. because some .png images are input mask. If you delete all .png images, the code could not find the input mask.
